module/userProjects/helper.js

```diff
@@ -2152,88 +2180,29 @@ module.exports = class UserProjectsHelper {
    * List of projects.
    * @method
    * @name list
-   * @param {String} userId - Logged in user id.
-   * @param {Number} pageNo - Page number.
-   * @param {Number} pageSize - Page size.
-   * @param {String} filter - discoveredByMe or createdByMe.
    * @returns {Array} List of projects.
    */
   
-    static list( userId, pageNo, pageSize, searchText = "", filter) {
-        return new Promise(async (resolve, reject) => {
-            try {
-
-                let query = {
-                    isDeleted : false,
-                    userId : userId
-                }
-                // If 'searchText' is provided, create a search query using '$or'.
-                if (searchText !== "") {
-                    queryObject["$or"] = [
-                        { externalId: new RegExp(searchText, "i") },
-                        { title: new RegExp(searchText, "i") },
-                        { description: new RegExp(searchText, "i") }
-                    ];
-                }
-                if (filter === CONSTANTS.common.DISCOVERED_BY_ME) {
-                    // Include records with projectTemplateId
-                    query.projectTemplateId = { $exists: true }; 
-                } else if (filter === CONSTANTS.common.CREATED_BY_ME) {
-                    // Include records without projectTemplateId
-                    query.projectTemplateId = { $exists: false }; 
-                }
-                
-                
-                let projects = await projectQueries.projectDocument(
-                    query
-                );
-
-                // Calculate the indices for pagination.
-                const startIndex = (pageNo - 1) * pageSize;
-                const endIndex = pageNo * pageSize;
-
-                // Slice the 'projects' array to get paginated results.
-                const paginatedResults = projects.slice(startIndex, endIndex);
+  static list( bodyData ) {
+    return new Promise(async (resolve, reject) => {
```

Comment:
@Prajwal17Tunerlabs we cannot remove this api

Reply:
i didnt get you, its not removed, it is still present in the code